assembly/__tests__/env-opdatastore.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { env } from '../env/index';
+
+import { changeCallStack, resetStorage } from '../vm-mock/storage';
+import { Args, bytesToString, stringToBytes } from '@massalabs/as-types';
+
+describe('Testing mocked Operation Datastore', () => {
+  test('hasOpKey - key not exists', () => {
+    expect(env.hasOpKey(stringToBytes('key1'))).toBeNull();
+  });
+
+  test('getOpKey - key exists', () => {
+    env.set(stringToBytes('key1'), stringToBytes('value1'));
+    expect(env.hasOpKey(stringToBytes('key1'))).not.toBeNull();
+    resetStorage();
```

Comment:
you have a beforeEach function in as-pect. Maybe you could use it for resetStorage

Reply:
Nice! Didn't know about that, actually done using 'afterEach' to be sure Storage is reset after this test group.

Reply:
perfect